docs/NOW-spec.md

```diff
@@ -1412,11 +1384,11 @@ The NOW_EXEC_PROCESS_MSG message is used to send a Windows [CreateProcess()](htt
 
 **sessionId (4 bytes)**: A 32-bit unsigned integer containing a unique remote execution session id.
 
-**filename (variable)**: A NOW_STRING structure containing the file name. Corresponds to the lpApplicationName parameter.
+**filename (variable)**: A NOW_VARSTR structure containing the file name. Corresponds to the lpApplicationName parameter.
```

Comment:
There is no `NOW_STRING` in this specification (section empty). I think `NOW_VARSTR` suits perfect here